spec/configuration_store_spec.lua

```diff
@@ -11,9 +12,56 @@ describe('Configuration Store', function()
 
       assert.equal(service, store:find_by_id('42'))
     end)
+
+    it('sets the configured flag to true', function()
+      local store = configuration.new()
+
+      store:store({ services = { id = '42', hosts = { 'example.com' } } })
+      assert.truthy(store.configured)
+    end)
+
+    describe('when path routing is enabled', function()
+      setup(function() -- Wouldn't be needed if we injected path_routing_enabled
+        package.loaded.configuration_store.path_routing = true;
```

Comment:
What about using `configuration` instead? What require returns should be the same object.